frontend/app/remark.tsx

```diff
@@ -14,6 +14,7 @@ import { getConfig } from 'common/api';
 import { fetchHiddenUsers } from 'store/user/actions';
 import { restoreCollapsedThreads } from 'store/thread/actions';
 import parseQuery from 'utils/parseQuery';
+import { parseBooleansFromDictionary } from 'utils/parseBooleansFromDictionary';
```

Comment:
[Our developer guide](https://github.com/umputun/remark42/blob/master/frontend/README.md) is confusign, but:
> any other files should be named also in kebab-case. For example ./app/utils/get-param.ts